src/controllers/employee.ts

```diff
@@ -0,0 +1,62 @@
+import { Request, RequestHandler, Response } from "express";
+import jwt from "jsonwebtoken";
+import { config } from "../config/config";
+import { EmployeeService } from "../services/employee";
+import { CreateEmployee } from "../types/employee";
+
+const signup: RequestHandler = async (req: Request, res: Response) => {
+    const createEmployee: CreateEmployee = req.body
+    try {
+        const employee = await EmployeeService.signup(createEmployee)
+        if (employee)
```

Comment:
please don't make curly braces and ; optional, make as compulsory. It will make code more readable.